### ISSUE TITLE
Charts dashlets on 2 and 3 column tabs do not scale

### ISSUE THREAD

**Issue by maladrill** [salesagility/SuiteCRM#3762]
#### Issue
When you add chart to dashboard tab with more than one column it does not scale to fit into the column.

#### Actual Behavior
Chart takes as much space as if it was only one column tab.

![tabs](https://user-images.githubusercontent.com/12087022/27390798-826e4a9c-56a2-11e7-9b21-b5db88008efd.PNG)

#### Steps to Reproduce
Create dashboard chart with 2 columns. 
Add any chart (tested on pipeline by stage and outcome by month charts)
Add any dashlet and move it to the second column.

#### Your Environment
Tested on https://demo.suiteondemand.com 
Chrome 58.0.3029.110 (64-bit)
Version 7.9.0
Reproduced on 7.9.1

**Comment by gunnicom** [comment]
Can you check if this fixes it for you (i use this in my 7.8.4).
In modules/AOR_Charts/AOR_Chart.php around line 262 there is:
`$defaultWidth = 900;`
I changed it to:
`$defaultWidth = "";`

**Comment by maladrill** [comment]
No, it did not help. I edited the variable, run Quick Repair and Rebuild, created new Tab and added dashlets. 
The result is, that the home page is not widened outside of the layout as before (on screenshot below there is no horizontal scrollbar which you can see in the screen above), but chart still takes 3/4 of the dashboard.
![charts_fix](https://user-images.githubusercontent.com/12087022/27426188-1de7e4fe-573b-11e7-9420-2da748fca241.PNG)

**Comment by gunnicom** [comment]
Hmm, maybe i got other changes here, or it behaves different in SuiteP theme.

**Comment by allysonsouza** [comment]
I'm having the same issue, my solution at the moment is to add a 1 column tab and use charts dashlets only on that column.

**Comment by johnM2401** [comment]
Hey!
Thank you for raising this!

This appears to have since been resolved.
I've given this a try in the latest release and have not been able to replicate this.

My charts are remaining in Column 1, where set:
<img width="1611" height="799" alt="Image" src="https://github.com/user-attachments/assets/c5fbe12a-efb2-4676-baea-3bcbae3f156d" />

I'm going to close this off for now.

However, if you are still seeing this issue in the latest releases of SuiteCRM, please don't hesitate to get in touch.

Thank you for your contributions to the project!
